docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -1158,78 +1206,75 @@ Your application uses this TLS registry extension to resolve ACME protocol chall
+You can use this domain name to create a Let's Encrypt account and pass the Let's Encrypt ACME challenges to prove that you own this domain.
+You can use https://ngrok.com/[ngrok] to start experimenting with the Quarkus Let's Encrypt ACME feature; for more information, see the <<lets-encrypt-ngrok,Testing with ngrok>> section below.
 
-[source, properties]
+. Your Quarkus HTTPS application must use a _build-time_ property to enable a Let's Encrypt ACME challenge route:
++
+[source,properties]
 ----
 quarkus.tls.lets-encrypt.enabled=true
 ----
-
++
 The TLS registry can manage the challenge process from either the main HTTP interface or the management interface.
 Using a management interface is **strongly** recommended to let Quarkus deal with ACME challenge configuration separately from the main application's deployment and security requirements:
-
-[source, properties]
++
+[source,properties]
 ----
 quarkus.tls.lets-encrypt.enabled=true
 quarkus.management.enabled=true
 ----
-
++
 The challenge is served from the primary HTTP interface (accessible from your DNS domain name).
-
++
 IMPORTANT: Do not start your application yet.
 
-=== Application preparation
-
-Before you request a Let's Encrypt certificate, you must run the TLS registry Let's Encrypt CLI `prepare` command to prepare your application:
-
-[source, shell]
+. Before you request a Let's Encrypt certificate, you must run the TLS registry Let's Encrypt CLI `prepare` command to prepare your application:
++
+[source,shell]
 ----
 quarkus tls lets-encrypt prepare --domain=<domain-dns-name>
 ----
-
++
 IMPORTANT: Make sure you run a prepare command in the root directory of your application.
-
++
 The `prepare` command does the following:
-
++
 * Creates a `.letsencrypt` folder in your application's root directory
-* Creates a self-signed domain certificate and private key for your application configured in the previous <<lets-encrypt-prerequisites>> step to be able to start and accept HTTPS requests
+* Creates a self-signed domain certificate and private key for your application configured in the previous <<lets-encrypt-prerequisites,Let's Encrypt prerequisites>> step to be able to start and accept HTTPS requests
 * Creates a `.env` configuration file in your application's root directory and configures the application to use the self-signed domain certificate and private key (until we get the Let's Encrypt certificate)
-
++
 The following snippet shows an example of the generated `.env` file:
-
-[source, properties]
++
+[source,properties]
 ----
 quarkus.tls.key-store.pem.acme.cert=.letsencrypt/lets-encrypt.crt
 quarkus.tls.key-store.pem.acme.key=.letsencrypt/lets-encrypt.key
 ----
-
++
 NOTE: The `.env` file does not contain the `quarkus.tls.lets-encrypt.enabled` and `quarkus.management.enabled` properties as they are build-time properties that require a rebuild of the application.
 
-=== Start your application
-
-You can start your application:
+.Procedure
```

Comment:
`Procedure` reads a bit unusual

Reply:
Well, it is a way RHBQ docs are being written down - to rewrite the text so that headings are not needed if they can be replaced by a procedure with steps.
Besides, `=== Start your application` is not a great heading :))